modules/internal/ChapelBase.chpl

```diff
@@ -21,134 +21,18 @@
-    compilerWarning("CHPL_TARGET_PLATFORM not set");  }
-
-  config param CHPL_HOST_COMPILER = "unset";
-  if (CHPL_HOST_COMPILER == "unset") {
-    compilerWarning("CHPL_HOST_COMPILER not set");
-  }
-
-  config param CHPL_TARGET_COMPILER = "unset";
-  if (CHPL_TARGET_COMPILER == "unset") {
-    compilerWarning("CHPL_TARGET_COMPILER not set");
-  }
-
-  config param CHPL_TARGET_ARCH = "unset";
-  if (CHPL_TARGET_ARCH == "unset") {
-    compilerWarning("CHPL_TARGET_ARCH not set");
-  }
-
-  config param CHPL_LOCALE_MODEL = "unset";
-  if (CHPL_LOCALE_MODEL == "unset") {
-    compilerWarning("CHPL_LOCALE_MODEL not set");
-  }
-
-  config param CHPL_COMM = "unset";
-  if (CHPL_COMM == "unset") {
-    compilerWarning("CHPL_COMM not set");
-  }
-
-  config param CHPL_COMM_SUBSTRATE = "unset";
-  if (CHPL_COMM_SUBSTRATE == "unset") {
-    compilerWarning("CHPL_COMM_SUBSTRATE not set");
-  }
-
-  config param CHPL_GASNET_SEGMENT = "unset";
-  if (CHPL_GASNET_SEGMENT == "unset") {
-    compilerWarning("CHPL_GASNET_SEGMENT not set");
-  }
-
-  config param CHPL_TASKS = "unset";
-  if (CHPL_TASKS == "unset") {
-    compilerWarning("CHPL_TASKS not set");
-  }
-
-  config param CHPL_THREADS = "unset";
-  if (CHPL_THREADS == "unset") {
-    compilerWarning("CHPL_THREADS not set");
-  }
-
-  config param CHPL_LAUNCHER = "unset";
-  if (CHPL_LAUNCHER == "unset") {
-    compilerWarning("CHPL_LAUNCHER not set");
-  }
-
-  config param CHPL_TIMERS = "unset";
-  if (CHPL_TIMERS == "unset") {
-    compilerWarning("CHPL_TIMERS not set");
-  }
-
-  config param CHPL_MEM = "unset";
-  if (CHPL_MEM == "unset") {
-    compilerWarning("CHPL_MEM not set");
-  }
-
-  config param CHPL_MAKE = "unset";
-  if (CHPL_MAKE == "unset") {
-    compilerWarning("CHPL_MAKE not set");
-  }
-
-  config param CHPL_ATOMICS = "unset";
-  if (CHPL_ATOMICS == "unset") {
-    compilerWarning("CHPL_ATOMICS not set");
-  }
-
-  config param CHPL_NETWORK_ATOMICS = "unset";
-  if (CHPL_NETWORK_ATOMICS == "unset") {
-    compilerWarning("CHPL_NETWORK_ATOMICS not set");
-  }
-
-  config param CHPL_GMP = "unset";
-  if (CHPL_GMP == "unset") {
-    compilerWarning("CHPL_GMP not set");
-  }
-
-  config param CHPL_HWLOC = "unset";
-  if (CHPL_HWLOC == "unset") {
-    compilerWarning("CHPL_HWLOC not set");
-  }
-
-  config param CHPL_REGEXP = "unset";
-  if (CHPL_REGEXP == "unset") {
-    compilerWarning("CHPL_REGEXP not set");
-  }
-
-  config param CHPL_WIDE_POINTERS = "unset";
-  if (CHPL_WIDE_POINTERS == "unset") {
-    compilerWarning("CHPL_WIDE_POINTERS not set");
-  }
-
-  config param CHPL_LLVM = "unset";
-  if (CHPL_LLVM == "unset") {
-    compilerWarning("CHPL_LLVM not set");
-  }
-
-  config param CHPL_AUX_FILESYS = "unset";
-  if (CHPL_AUX_FILESYS == "unset") {
-    compilerWarning("CHPL_AUX_FILESYS not set");
-  }
-
   // Is the cache for remote data enabled at compile time?
   config param CHPL_CACHE_REMOTE: bool = false;
 
   config param noRefCount = false;
-  
```

Comment:
This was brought to my attention earlier, and I reached the same conclusion. Will be more mindful of this in the future.